doc/stomp_planner/stomp_planner_tutorial.rst

```diff
@@ -0,0 +1,168 @@
+    
+   **>** *Save this file in the* **config** *directory of the moveit_config package*. Also make sure that the dimensionality of the `stddev` array parameter is the same as the number of joints present in the planning group name of your robot.
+
+#. Modify the **move_group.launch** file. Open the **move_group.launch** in the launch directory and change the ```pipeline``` parameter value to ```stomp``` as shown below: ::
+
+        .
+        .
+        .
+    <!-- move_group settings -->
+    <arg name="allow_trajectory_execution" default="true"/>
+    <arg name="fake_execution" default="false"/>
+    <arg name="max_safe_path_cost" default="1"/>
+    <arg name="jiggle_fraction" default="0.05" />
+    <arg name="publish_monitored_planning_scene" default="true"/>
+
+    <!-- Planning Functionality -->
+    <include ns="move_group" file="$(find myworkcell_moveit_config)/launch/planning_pipeline.launch.xml">
+      <arg name="pipeline" value="stomp" />
+    </include>
+        .
+        .
+        .
+
+Running the Demo
+----------------
+If you have the ``panda_moveit_config`` from the `ros-planning/panda_moveit_config <https://github.com/ros-planning/panda_moveit_config>`_ repository you should be able to simply run the demo: ::
+
+  roslaunch panda_moveit_config demo.launch
+
+Running STOMP with Obstacles in the Scene
++++++++++++++++++++++++++++++++++++++++++
+To run STOMP in an evironment with obstacles, you can run the sample python script:
+
+  :codedir:`collision_scene_example.py<chomp_planner/scripts/collision_scene_example.py>`.
+
+This scripts creates a cluttered scene with four ostacles or a simple scene with one obstacle depending on the argument given to the script. One can also change the position/size of the obstacles to change the scene. 
+
+To run the STOMP planner with obstacles, open two terminals. In the first terminal start RViz and wait for everything to finish loading: ::
+
+  roslaunch panda_moveit_config demo_stomp.launch
+
+In the second terminal, run either of the two commands: ::
+
+  rosrun moveit_tutorials collision_scene_example.py cluttered
+
+or: ::
+
+  rosrun moveit_tutorials collision_scene_example.py sparse
+
+Next, in RViz, select STOMP in the MotionPlanning pannel under the Context tab. Set the desired start and goal states by moving the end-effector around with the imarker and then click on the Plan button under the Planning tab in the MotionPlanning pannel to start planning. The planner will now attempt to find a feasible solution between the given start and end position. STOMP performs better than CHOMP in avoiding obstacles. This is due to its stochastoc nature which produces non-jerky trajectories which are often produced by CHOMP to avoid obstacles
```

Comment:
stochastoc -> stochastic

Rephrase the sentence; it reads now as if CHOMP is often producing non-jerky trajectories to avoid obstacles. I think you mean the opposite.

Reply:
rephrased so it sounds correct now.